.license_scout.yml

```diff
@@ -26,10 +26,10 @@ directories:
   - ./components/automate-prometheus
   - ./components/automate-ui
   - ./components/automate-ui-devproxy
-  - ./components/automate-workflow-ctl
-  - ./components/automate-workflow-web
-  - ./components/automate-workflow-nginx
-  - ./components/automate-workflow-server
+  # - ./components/automate-workflow-ctl
+  # - ./components/automate-workflow-web
+  # - ./components/automate-workflow-nginx
+  # - ./components/automate-workflow-server
```

Comment:
I think we want these turned on still?

Reply:
oh shit sorry
i did that locally and forgot to remove